pipes/rules/assembly.rules

```diff
@@ -77,8 +102,9 @@ rule orient_and_impute:
             which will be captured in this output.
     '''
     input:  
-        fasta      = config["tmp_dir"]+'/'+config["subdirs"]["assembly"]+'/{sample}.assembly1-trinity.fasta',
-        ref_genome = objectify_remote(expand( '{refDir}/'+'{ref_name}.fasta', refDir=config["ref_genome_dir"], ref_name="reference" ))
+        fasta      = config["tmp_dir"]+'/'+config["subdirs"]["assembly"]+'/{sample}.assembly1-trinity-spades.fasta',
```

Comment:
For this particular PR, let's not make the SPAdes changes the new default behavior in the pipeline quite yet? I'd propose changing this line to read:
```
fasta      = config["tmp_dir"]+'/'+config["subdirs"]["assembly"]+'/{sample}.assembly1-trinity.fasta',
```
... which would keep the new `assemble_spades` command available in the CLI, keep the new `assemble_spades` rule available in snakemake, but that rule wouldn't trigger yet for any workflow, since no downstream rules would utilize it (yet). Then let's kick it around a bit more and, when we feel good about it, submit a minimal PR that makes it the default in the pipeline.

Rest of the gapfill changes in this rule look good.